docs/source/installation.mdx

```diff
@@ -23,12 +23,10 @@ pip install bitsandbytes
 
 ```bash
 git clone https://github.com/TimDettmers/bitsandbytes.git && cd bitsandbytes/
-CUDA_VERSION=XXX make cuda12x
-python setup.py install
+cmake -B build -DBUILD_CUDA=ON -S .
+python setup.py develop
```

Comment:
```suggestion
python setup.py install
```

Reply:
`pip install .` – `setup.py` shouldn't needed to be run by hand.

Reply:
ok ! modified it